modules/tinymce/src/core/main/ts/annotate/AnnotationFilter.ts

```diff
@@ -15,14 +15,13 @@ import * as Markings from './Markings';
 const setup = (editor: Editor, registry: AnnotationsRegistry): void => {
   const identifyParserNode = (span: AstNode): Optional<AnnotatorSettings> => Optional.from(span.attr(Markings.dataAnnotation())).bind(registry.lookup);
 
-  editor.on('init', () => {
```

Comment:
For others reading this, I'm assuming this was removed since by the time the annotator is setup the serializer is already initialised so this wasn't needed.

Reply:
Yes, and also it should not be added that late in the process it should at least be preinit. Init is more when the editor content is done in theory some one could call editor.getContent at that stage to get the initial snapshot out but I guess the core inits will fire before plugin or custom inits but anyway.